elyra/metadata/schemas/airflow.json

```diff
@@ -118,6 +118,16 @@
             "category": "Cloud Object Storage"
           }
         },
+        "user_namespace": {
+          "title": "Apache Airflow User Namespace",
+          "description": "The Apache Airflow user namespace used to run DAG workflows",
+          "type": "string",
+          "pattern": "^[a-z0-9][-a-z0-9]*[a-z0-9]$",
+          "maxLength": 63,
+          "uihints": {
+            "category": "Kubernetes"
```

Comment:
Could you make sure the category is correct? KFP Schema is using "Kubeflow Pipelines" and has no "Kubernetes" category. Note that the category is the section on the UI where this property is grouped and will be displayed.

Reply:
Yes, in this case the category should be `Apache Airflow`, even though it's a Kubernetes related setting, to be consistent how it was defined for KFP.

Reply:
Hmm - since `cos_secret` is in the `Kubernetes` category for each (KFP and Airflow), perhaps it would be better to update the kfp schema's `user_namespace` category instead?

Reply:
felt that since the user_namespace was more in line with kubernetes, I opted to put that into a kubernetes category like the cos_secret PR. If we aren't feeling it, I've moved it back to align with the current position in kfp schema and added screens below